autoscheduler/frontend/src/tests/ui/SchedulingPage.test.tsx

```diff
@@ -10,11 +10,67 @@ import { Provider } from 'react-redux';
 import {
   render, fireEvent, waitFor, queryByText as queryContainerByText,
 } from '@testing-library/react';
+import * as router from '@reach/router';
 import autoSchedulerReducer from '../../redux/reducer';
 import SchedulingPage from '../../components/SchedulingPage/SchedulingPage';
 import { mockFetchSchedulerGenerate } from '../testData';
 
 describe('Scheduling Page UI', () => {
+  // setup and teardown spy function on navigate
+  const navSpy = jest.spyOn(router, 'navigate');
+  // don't actually call navigate
+  beforeAll(() => navSpy.mockImplementation(() => null));
+  // restore navigate to original
+  afterAll(navSpy.mockRestore);
+
+  describe('redirects to the homepage', () => {
+    test('when no term is selected', async () => {
+      // arrange
+      const store = createStore(autoSchedulerReducer);
+
+      // sessions/get_last_term
+      fetchMock.mockResponseOnce(JSON.stringify({}));
+
+      // render
+      render(
+        <Provider store={store}>
+          <SchedulingPage />
+        </Provider>,
+      );
+
+      // do nothing - redirects automatically
```

Comment:
typically in these scenarios instead of saying "do nothing", we'd instead consider the "act" part of the test rendering, so you can remove the `do nothing` line and change `render` -> `act.

Seeing the
```
// render
render(...)
```
made me think of a pic I saw on reddit today (frontpage of /r/programmerhumor):
![image](https://user-images.githubusercontent.com/7295783/95799712-23902900-0cbb-11eb-8534-902d1f2cf2c8.png)

Reply:
Cool thanks! I changed the comments a bit.